.eslintrc.json

```diff
@@ -8,5 +8,8 @@
   ],
   "env": {
     "jest": true
+  },
+  "rules": {
+    "no-use-before-define": ["error", { "functions": false }]
```

Comment:
Did you add this rule because of the `index.js`?

Reply:
This is an issue for almost every function call in this package, since I want to keep an order from most-significant functions on top to least-significant helper functions on the bottom.

Disabling this rule seems to be safe (from the [eslint-rules-docs](https://eslint.org/docs/rules/no-use-before-define)):
> Function declarations are hoisted, so it's safe